collections/index.html

```diff
@@ -34,6 +34,8 @@
 
 <h2>All Collections</h2>
 
+<p>Collections are curated stories showcasing the best of our engineers, experiments, tools, and resources.</p>
```

Comment:
```suggestion
<p>Collections are curated stories showcasing the best of our team, experiments, tools, and resources.</p>
```

Reply:
![image](https://github.com/OpenINF/openinf.github.io/assets/17770407/535cb052-504a-4b88-85c4-b82aebe498a8)

Engineers! 🚀 (form spotted w/o choice given &mdash;

&mdash;more free trials of emerging tech&mdash;

&mdash;more relevant business salespeople willing to chat too?)

Reply:
the very next page see after submitting 😅  (this is kinda a pet peeve of mine at this point);


![image](https://github.com/OpenINF/openinf.github.io/assets/17770407/70f2a364-f1cf-4074-ab5e-d55cf2515741)

if i had engineer in my bio and someone referred to me as lowly dev immediately after identifying as Engineer i would not be giving them my business